docs/data/migration/migration-pickers-v6/migration-pickers-v6.md

```diff
@@ -290,6 +290,13 @@ The string argument of the `dayOfWeekFormatter` prop has been replaced in favor
  />
 ```
 
+### Strict typing of the date-related props
+
+All the date-related props are now strictly typed to only accept the date format supported by your adapter
+(`Date` object for `date-fns`, `daysjs.Dayjs` object for `days-js`, etc.).
+
+You can learn more about this change in the [dedicated doc section](/x/react-date-pickers/base-concepts/#typing-of-the-date).
```

Comment:
You can also consider using the format suggested by Sam on how to name the links to [other doc sections](https://mui.com/x/react-date-pickers/date-field/#customize-the-date-format).
```suggestion
:::info
See [Base concepts—Typing of the date](/x/react-date-pickers/base-concepts/#typing-of-the-date) for more details.
:::
```

Reply:
Would you apply the same rational to the other "You can learn more" of the page?

Reply:
It's just a suggestion, but especially in this section, I think that it would improve the visibility and importance of the link.
As for the other sections—it's up to you. Technically it's out of the scope of this issue, but I don't think it would hurt. 👌

Reply:
It's out of the scope but it's a super simple change that is not worth a dedicated review IMHO, I'll do the switch :ok_hand:

Reply:
Done :+1:

Reply:
I will do a follow up for the rest of the doc if I find some time